cdc/sink/manager.go

```diff
@@ -145,8 +146,14 @@ func (m *Manager) destroyTableSink(ctx context.Context, tableID model.TableID) e
 	return m.backendSink.Barrier(ctx, tableID)
 }
 
-func (m *Manager) getCheckpointTs() uint64 {
-	return atomic.LoadUint64(&m.checkpointTs)
+func (m *Manager) getCheckpointTs(tableID model.TableID) uint64 {
+	checkPoints, ok := m.tableCheckpointTsMap.Load(tableID)
+	if ok {
+		return checkPoints.(uint64)
+	}
+	// cannot find table level checkpointTs because of no table level resolvedTs flush task finished successfully,
+	// for example: first time to flush resolvedTs but cannot get the flush lock, return changefeed level checkpointTs is safe
+	return m.changeFeedCheckpointTs
```

Comment:
When do we update the `changeFeedCheckepoinTs`? Will it cause checkpoint regression on grafana dashboard? Will it cause owner sending a regressed checkpoint to downstream?

Reply:
Now we do not update the `changeFeedCheckepoinTs`,  the sinkNode will not update its checkpointTs if the returned value is smaller than it. and we can update it per tick if needed.
grafana and owner get a table's checkpointTs from sinkNode,  if I understand correctly,   grafana and the owner will not be impacted.